Network/Attributes/PacketRequestAttribute.cs

```diff
@@ -27,31 +28,49 @@
 //  You should have received a copy of the GNU Lesser General Public License
 //  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // ***********************************************************************
+
 #endregion Licence - LGPLv3
+
+using Network.Packets;
 using System;
 
 namespace Network.Attributes
 {
     /// <summary>
-    /// In order to generate an answer out of a question, the relations
-    /// between the request and response packets have to be set.
-    /// E.g NameResponse -> NameRequest.
+    /// Maps a request packet to the response packet that handles it. This
+    /// attribute should be placed on the response packet (must inherit from
+    /// <see cref="ResponsePacket"/> and the <see cref="Type"/> of the
+    /// <see cref="RequestPacket"/> that it
+    /// handles should be given.
     /// </summary>
     public class PacketRequestAttribute : Attribute
     {
+        #region Properties
+
         /// <summary>
-        /// Initializes a new instance of the <see cref="PacketRequestAttribute"/> class.
+        /// The <see cref="Type"/> of the <see cref="RequestPacket"/> that the
+        /// <see cref="ResponsePacket"/> handles.
         /// </summary>
-        /// <param name="type">The type.</param>
+        public Type RequestType { get; }
```

Comment:
Order is wrong.

1. Variables
2. Constructors
3. Properties
4. Methods

Reply:
Dont agree here at all. Surely properties should come before the constructor and after the variables, as they are similar to the variables more than to the methods? Granted, behind the scenes they are simply method calls but at the same time, they are a method of storing data. IMHO, a programmer should know what data an object stores and manipulates before seeing any actual manipulation of said data. This needs to be discussed further.